blog/closed-beta.md

```
@@ -13,6 +13,21 @@ Announcing our Closed Beta, offering market efficiency and flexibility for on-ch
 ![](../static/img/posts/closed-beta.png)
 
 
+Loopscale is blockchain infrastructure that brings efficiency and flexibility to asset-based capital markets. It enables the compatibility of complex and long-tail collateral as well as configurability of loan structure. The Loopscale platform facilitates borrowing transactions ranging from DePIN participants financing hardware purchases with networ---
+title: Access the Loopscale Closed Beta
+# description: Announcing our Closed Beta, offering market efficiency and flexibility for on-chain capital markets via fixed-rate borrowing, diverse collateral support, and more.
```

markdownlint-cli2 (0.17.2)

Trailing punctuation in heading
Punctuation: '.'

(MD026, no-trailing-punctuation)